ansible/configs/open-environment-azure/post_software.yml

```diff
@@ -36,19 +37,39 @@
         - "Subscription ID: {{ azure_subscription_id }}"
         - ""
         - "Azure CLI quickstart:"
-        - "GUID={{ guid }}"
-        - "CLIENT_ID={{ azapp['applications'][0].app_id }}"
-        - "PASSWORD={{ azpass }}"
-        - "TENANT={{ azure_tenant }}"
-        - "SUBSCRIPTION={{ azure_subscription_id }}"
-        - "RESOURCEGROUP=openenv-{{ guid }}"
+        - "export GUID={{ guid }}"
+        - "export CLIENT_ID={{ azapp['applications'][0].app_id }}"
+        - "export PASSWORD={{ azpass }}"
+        - "export TENANT={{ azure_tenant }}"
+        - "export SUBSCRIPTION={{ azure_subscription_id }}"
+        - "export RESOURCEGROUP=openenv-{{ guid }}"
```

Comment:
i think you can 

```yaml
agnosticd_user_info:
  msg:
     - line1
     - line2
     - ...
```
directly, instead of calling it N x times.   https://github.com/redhat-cop/agnosticd/blob/development/ansible/action_plugins/agnosticd_user_info.py#L66-L67

Reply:
fixed.